docs/develop/4-specifications/01_index.md

```diff
@@ -0,0 +1,20 @@
+---
+id: index
+title: Index
```

Comment:
I would call it like the menu entry "Technical Specification"

Reply:
Yeah this is supposed to be the entry page for all specifications, but for now it only contains links in the same page. Anyway, I removed the navbar (for now) and changed the page name: https://github.com/KILTprotocol/docs/pull/95/commits/5718b309614acfe35b0bd43027a6b9a1a2e4e20d